### PR TITLE
feat(Naver): rewrite Presence

### PR DESCRIPTION
**Pull request by Slowlife01**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image
image

</details>

### CODE REVIEW

File: websites/N/Naver/presence.ts
Comment (Slowlife01) [inline websites/N/Naver/presence.ts:1]: Not sure if this would actually work..

File: websites/N/Naver/presence.ts [hunk 1]
```diff
@@ -121,52 +75,15 @@ presence.on("UpdateData", async () => {
...
-				};
-
-			if (video?.paused) {
-				delete tempData.startTimestamp;
-				delete tempData.endTimestamp;
-
-				tempData.smallImageKey = assets[`${data.service.toLowerCase()}_pause`];
-				tempData.smallImageText = "Paused";
-			}
+	if (location.hostname.includes("chzzk")) return;
```
Comment (Slowlife01) [inline websites/N/Naver/presence.ts:78]: Not sure if this would actually work..

File: websites/N/Naver/presence.ts [hunk 2]
```diff
@@ -1,114 +1,68 @@
...
+		naver_series:
+			"https://cdn.discordapp.com/app-icons/907151903564120135/a7f2fb845d4b281b2c4dc34dd494969c.png?size=512",
+		naver_tv:
+			"https://cdn.discordapp.com/app-icons/827170810870890517/d23dc00b5edfade991b32fdd8cd6faa6.png?size=512",
+		naver_papago:
+			"https://cdn.discordapp.com/app-icons/827170714729578546/b3f02012b4ae188f245d9cd4e78be721.png?size=512",
+		naver_blog:
+			"https://cdn.discordapp.com/app-icons/827171204288872449/92aa96637c18dab3a2a8866d09beff24.png?size=512",
+		naver_blog_browse:
+			"https://cdn.discordapp.com/app-assets/827171204288872449/827173061572296804.png",
```
Comment (theusaf) [inline websites/N/Naver/presence.ts:31]:
```suggestion
			"https://cdn.discordapp.com/app-assets/827171204288872449/827173061572296804.png?size=512",
```

File: websites/N/Naver/presence.ts [hunk 3]
```diff
@@ -1,114 +1,68 @@
...
 	};
 /* eslint-enable camelcase */
 
+const enum MainAssets {
+	Browse = "https://cdn.discordapp.com/app-assets/825307070584586250/826435062756278273.png",
+	Book = "https://cdn.discordapp.com/app-assets/825307070584586250/826447462554140704.png",
+	Play = "https://cdn.discordapp.com/app-assets/827170810870890517/827172409979305995.png",
+	Pause = "https://cdn.discordapp.com/app-assets/827170810870890517/827172409936314378.png",
+	Language = "https://cdn.discordapp.com/app-assets/827170714729578546/827172528383590460.png",
+}
```
Comment (theusaf) [inline websites/N/Naver/presence.ts:43]: Assets should be 512x512px
```suggestion
const enum MainAssets {
	Browse = "https://cdn.discordapp.com/app-assets/825307070584586250/826435062756278273.png?size=512",
	Book = "https://cdn.discordapp.com/app-assets/825307070584586250/826447462554140704.png?size=512",
	Play = "https://cdn.discordapp.com/app-assets/827170810870890517/827172409979305995.png?size=512",
	Pause = "https://cdn.discordapp.com/app-assets/827170810870890517/827172409936314378.png?size=512",
	Language = "https://cdn.discordapp.com/app-assets/827170714729578546/827172528383590460.png?size=512",
}
```

File: websites/N/Naver/presence.ts [hunk 4]
```diff
@@ -176,460 +91,169 @@ presence.on("UpdateData", async () => {
...
+						presenceData.smallImageKey = MainAssets.Pause;
+						presenceData.smallImageText = "Paused";
+						delete presenceData.startTimestamp;
+						delete presenceData.endTimestamp;
+					}
 
 					presenceData.buttons = [
 						{
-							url: document.baseURI,
+							url: location.href,
```
Comment (theusaf) [inline websites/N/Naver/presence.ts:139]: For consistency with other presences, use `document.location` properties.
```ts
const { pathname, href } = document.location;
{ url: href, label: "..." }
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by theusaf — APPROVED** [review]

**Review by Timeraa — APPROVED** [review]

**Pull request merged**